### PR TITLE
Version bump tsunami (but primarily Python 3.9)

### PR DESCRIPTION
**Pull request by mathcolo**
⚠️ For after bus goes in!

- Python 3.9
- node 16
- Make sure Pipfile and requirements.txt match

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "lint-backend": "cd server && pipenv run flake8 --ignore=E501,E731 --exclude .chalice",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "postinstall": "cd server && pipenv install --python=3.7"
+    "postinstall": "cd server && pipenv install --dev"
```
Comment (mathcolo) [inline package.json:34]: @nathan-weinberg Unfortunately `--python=3.7` causes every `pipenv install` to delete and re-create the virtualenv, so I removed it

Comment (nathan-weinberg) [reply package.json:34]: Gotcha, that should be fine if it worked for you locally :+1:

**Review by nathan-weinberg — APPROVED** [review]
Assuming this was all tested LGTM, thanks for doing this @mathcolo!

**Comment by mathcolo** [comment]
@nathan-weinberg nice ^ lol

**Comment by mathcolo** [comment]
Closing because we're going to do python and js parts separately per @austinjpaul

**Pull request closed without merging**